### PR TITLE
docs(ecosystem): add @clerk/fastify plugin

### PR DESCRIPTION
**Pull request by dimkl**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by Fdawgs** [comment]
@dimkl please see the CI failures and rectify, and we can get this added. 😄

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -635,7 +635,10 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`@clerk/fastify`](https://github.com/clerkinc/javascript/tree/main/packages/fastify) Clerk is the easiest way to add authentication and user management to your Fastify application.
```
Comment (Uzlopak) [inline docs/Guides/Ecosystem.md:638]: Can we please not have advertisement showed down our throat? Is Clerk really the easiest way to add authentication and user management to fastify?

Comment (jsumners) [inline docs/Guides/Ecosystem.md:638]:
```suggestion
- [`@clerk/fastify`](https://github.com/clerkinc/javascript/tree/main/packages/fastify) Clerk is an easy way to add authentication and user management to your Fastify application.
```

Comment (dimkl) [reply docs/Guides/Ecosystem.md:638]: I just reused the existing Readme from the clerk repo but I'll be more than happy to change accordingly 😅 .

Comment (dimkl) [reply docs/Guides/Ecosystem.md:638]: thank you for the suggestion but i will change it to be more generic and to be less than 80 chars (as linting rules indicate).

File: docs/Guides/Ecosystem.md [hunk 2]
```diff
@@ -635,7 +637,9 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+
```
Comment (Uzlopak) [inline docs/Guides/Ecosystem.md:640]:
```suggestion
```

File: docs/Guides/Ecosystem.md [hunk 3]
```diff
@@ -635,7 +637,9 @@ section.
...
 #### [Community Tools](#community-tools)
+
```
Comment (Uzlopak) [inline docs/Guides/Ecosystem.md:642]:
```suggestion
```

File: docs/Guides/Ecosystem.md (hunk 2 above)
Comment (dimkl) [reply docs/Guides/Ecosystem.md:640]: I thought that `Community Tools` should be the same as `Community` and `Core` headings (that have empty lines around) but i will drop them as suggested.

**Review by Uzlopak — APPROVED** [review]
Lgtm

**Review by Fdawgs — APPROVED** [review]

**Comment by dimkl** [comment]
Do we have an ETA of this being available in the https://www.fastify.io/ecosystem/ ?

**Comment by Uzlopak** [comment]
@Eomm 
?

**Comment by climba03003** [comment]
Seems like CircleCI is dead for at least 2 weeks.
I don't know who have privilege for it.
cc @mcollina @lmammino

**Pull request merged**